benchopt/stopping_criterion.py

```diff
@@ -212,7 +211,9 @@ def should_stop(self, stop_val, objective_list):
         n_eval = len(objective_list) - 1
         objective = objective_list[-1][self.key_to_monitor]
         delta_objective = self._prev_objective - objective
-        delta_objective /= abs(objective_list[0][self.key_to_monitor])
+        next_objective = objective_list[0][self.key_to_monitor]
```

Comment:
Why `next_objective`?

Reply:
to avoid calling lengthy `objective_list[0][self.key_to_monitor]` twice. I can remove it if you want.